login.css

```
@@ -1,3 +1,137 @@
+@import url('https://fonts.googleapis.com/css?family=Nunito');
+
 body {
-    background-color: dodgerblue;
+	height: 100vh;
+	font-family: 'Nunito', sans-serif;
+	background-color: dodgerblue;
+}
+
+.wrapper {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+
+.content {
+	width: 100%;
+	max-width: 500px;
+	text-align: center;
+	padding: 40px 15px 50px 15px;
+	border-radius: 5px;
+	background-color: #fff;
+}
+
+h1,
+h2 {
+	font-weight: 500;
+}
+
+header {
+	display: flex;
+	margin-bottom: 10px;
+}
+
+.image-container {
+	flex-basis: 20%;
+}
+
+.welcome-container {
+	flex-basis: 60%;
+	justify-content: center;
+	padding-top: 10px;
+	margin-bottom: 10px;
+}
+
+img {
+	width: 80px;
+	height: 80px;
+}
+
+.login-form {
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.input-group {
+	width: 100%;
+	max-width: 300px;
+	display: flex;
+	flex-direction: column;
+}
+
+.input-group input:first-child {
+	margin-bottom: 10px;
+}
+
+input {
+	font-size: 14px;
+	padding: 15px;
+	border-radius: 5px;
+	border: 1px solid #e4eff8;
+	background-color: #e4eff8;
+}
+
+input:hover {
+	background-color: #d7e2ec;
+}
+
+.btn-group {
+	width: 100%;
+	max-width: 300px;
+	display: flex;
+	margin: 20px 0;
+}
+
+button {
+	width: 50%;
+	font-size: 16px;
+	padding: 15px;
+	border-radius: 5px;
+	border: 1px solid #1b9fd8;
+	cursor: pointer;
+	background-color: #1b9fd8;
+}
+
```

```suggestion
button a {
	display: block;
	width: 100%;
	height: 100%;

	font-size: 16px;
	color: #ffffff;
	text-decoration: none;
}
```
Currently, the `a` inside the button is using the default `display: inline;`. Since it is an inline element, only clicking on the text in the button will link to index.html. This will make it so the whole button works. Also moving the id-agnostic properties here is cleaner.

---

I think better solution would be 'input type="button" wrapped inside a.